src/test/java/com/exasol/adapter/dynamodb/mapping/JsonMappingReaderTest.java

```diff
@@ -38,8 +39,7 @@ void testBasicMapping() throws IOException, AdapterException {
         final List<TableMapping> tables = schemaMapping.getTableMappings();
         final TableMapping table = tables.get(0);
         final List<ColumnMapping> columns = table.getColumns();
-        final List<String> columnNames = columns.stream().map(ColumnMapping::getExasolColumnName)
-                .collect(Collectors.toList());
+        final List<String> columnNames = getColumnNames(columns);
```

Comment:
Can you use there `Objects.requireNonNull()` instead of annotation?
```
final List<String> columnNames = Objects.requireNonNull(getColumnNames(columns));
```
This can skip the dependency.

Reply:
This is autogenerated when extracting a method. I think I just don't need the check for not null.